simtools/camera_efficiency.py

```diff
@@ -174,6 +177,7 @@ def simulate(self, force=False):
             file_simtel=self._file_simtel,
             file_log=self._file_log,
             label=self.label,
+            nsb_spectrum=self.config.nsb_spectrum,
```

Comment:
you added the nsb_spectrum here and I see that the zenith angle is already there. What about the azimuth? Doesn't it play a role in the simulation?